wordclock/settings.c

```diff
@@ -30,8 +30,8 @@ static const TSettings g_settings_default __attribute__((aligned(4))) = {
      "/get_with_age.php",  // hierbenikRequest
      0.0, // home lat
      0.0, // home lon
-     "rwassens.ddns.net",  // otaFwUrl
-     "10070",   // otaFwPort
+     "http://download.wssns.nl",  // otaFwUrl
```

Comment:
Wat heb je nu draaien op http://download.wssns.nl?
misschien kan je dat toevoegen aan de readme?

Reply:
daar staat nog geen release. Deze branch staat nu als dbg. Als dit gemerged is in devellop zet ik die als release,

Reply:
Bedoelde meer wat je hebt geïnstalleerd zodat de FW gedownload kan worden.

Reply:
Snap je vraag niet. Wssns.nl is mijn domein, gehost op een betaalde hosting.
Op de klok hebruik ik een standaard librarie voor http_ota download